src/mode/new.rs

```diff
@@ -0,0 +1,52 @@
+use std::{process::Command, env, path::Path, fs::File, io::Write};
+
+use clap::Parser;
+use anyhow::{anyhow, Result};
+
+use crate::mode;
+
+#[derive(Parser, Debug)]
+#[command(version)]
+pub struct Args {
+    #[arg(index = 1)]
+    pub new_args: Vec<String>,
+}
+
+fn add_lunatic_main_file() {
+    let text = format!(
+        "use std::time::Duration;
+use lunatic::{{sleep, spawn_link}};
+        
+fn main() {{
+    spawn_link!(|| println!(\"Hello, World! I'm a process.\"));
+    sleep(Duration::from_millis(100));
+}}"
+    );
+
+    let mut file = File::create("src/main.rs").expect("Opened src/main.rs");
```

Comment:
This is not portable. See https://doc.rust-lang.org/std/path/struct.PathBuf.html